cernopendata/templates/cernopendata_records_ui/records/detail.html

```diff
@@ -74,6 +74,14 @@ <h6 class="card-title">
                                 <h2>Description</h2>
                                 <p>{{record.abstract.description | safe}}</p>
                                 {{ link_list(record.abstract.links) }}
+                                {% if record.run_numbers %}
```

Comment:
@katilp We can see live about the best visual presentation of the run number information.  Currently it looks like:

![run_numbers](https://github.com/cernopendata/opendata.cern.ch/assets/517546/0dc3af19-ada7-4491-b54a-e52433bc36b4)

Reply:
I'd rather hide it by default and keep it in the metadata JSON only for now. 

It somehow takes importance from the validated run filter (the previous link) and - while it is valid information and certainly something we want to keep track of - it does not add information that users could or would actually use.

Reply:
OK, removed.  Just for information, this was the corresponding Jinja snippet:

```diff
diff --git a/cernopendata/templates/cernopendata_records_ui/records/detail.html b/cernopendata/templates/cernopendata_records_ui/records/detail.html
index 84ebd54f..1ba3bd02 100644
--- a/cernopendata/templates/cernopendata_records_ui/records/detail.html
+++ b/cernopendata/templates/cernopendata_records_ui/records/detail.html
@@ -74,6 +74,14 @@
                                 <h2>Description</h2>
                                 <p>{{record.abstract.description | safe}}</p>
                                 {{ link_list(record.abstract.links) }}
+                                {% if record.run_numbers %}
+                                  <p>This dataset was taken during the following run numbers:
+                                  {% for run_number in record.run_numbers %}
+                                     {{run_number}}
+                                     {{ "," if not loop.last else "" }}
+                                  {% endfor %}
+                                  </p>
+                                {% endif %}
                             </div>
                         </div>
                         {% endif %}
```